Tests/StoreKitUnitTests/PurchasesOrchestratorTests.swift

```diff
@@ -1137,6 +1137,7 @@ class PurchasesOrchestratorTests: StoreKitConfigTestCase {
         try AvailabilityChecks.iOS15APIAvailableOrSkipTest()
 
         self.customerInfoManager.stubbedCustomerInfoResult = .success(self.mockCustomerInfo)
+        self.mockStoreKit2TransactionListener?.mockResult = .init(.userCancelled)
```

Comment:
This test was failing after the changes to `MockStoreKit2TransactionListener` because it was returning the purchased transaction.